api/README.md

```diff
@@ -0,0 +1,8 @@
+```
+npm install
+npm run dev
+```
+
+```
+open http://localhost:3000
```

Comment:
This is nitpicky af, but since this isn't a command to be run, just leave it outside a codeblock